minerl/Malmo/Minecraft/Minecraft_Server.launch

```diff
@@ -1 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><launchConfiguration type="org.eclipse.jdt.launching.localJavaApplication"><stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="GradleStartServer"/><stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="Minecraft"/><stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="/Users/wguss/workspace/openai/minerl_data/minerl/minerl/env/Malmo/Minecraft/run"/><stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value=" -DFORGE_FORCE_FRAME_RECALC=true"/></launchConfiguration>
```

Comment:
What was the purpose of this file? Just an accidental configuration file for eclipse?

Reply:
Yes. Moreover, it was supposed to be included in the .gitignore, but due to a bug in the latter was not